File: examples/conditions-operator-string/conditions-operator-string.py
import json
import iam_floyd as statement

s = (
    # doc-start
    statement.Ec2() \
        .allow() \
        .to_start_instances() \
        .if_aws_request_tag('TagWithSpecialChars', '*John*', 'StringEquals')
    # doc-end
).to_json()

pretty = json.dumps(s, indent=4, sort_keys=True)
print(pretty)

File: examples/conditions-operator-string/conditions-operator-string.ts
import { deploy, out } from '../../helper/typescript/typescript_test';
import { statement } from '../../lib';

function getStatement() {
  return (
    // doc-start
    new statement.Ec2()
      .allow()
      .toStartInstances()
      .ifAwsRequestTag('TagWithSpecialChars', '*John*', 'StringEquals')
    // doc-end
  );
}
const s = [getStatement()];
out(s);
deploy(s);
